ViSearch.user.js

```diff
@@ -216,9 +226,250 @@ const width=800;
1},{"source":"tag34","target":"key4","value":1},{"source":"tag34","target":"key5","value":1},{"source":"tag34","target":"key7","value":1},{"source":"tag34","target":"key11","value":1},{"source":"tag34","target":"key16","value":1},{"source":"tag34","target":"key26","value":1},{"source":"tag34","target":"key27","value":1},{"source":"tag34","target":"key33","value":1},{"source":"tag34","target":"key35","value":1},{"source":"tag34","target":"key36","value":1},{"source":"tag34","target":"key45","value":1},{"source":"tag38","target":"key4","value":1},{"source":"tag38","target":"key7","value":1},{"source":"tag38","target":"key11","value":1},{"source":"tag38","target":"key16","value":1},{"source":"tag38","target":"key26","value":1},{"source":"tag38","target":"key27","value":1},{"source":"tag38","target":"key33","value":1},{"source":"tag38","target":"key35","value":1},{"source":"tag38","target":"key45","value":1},{"source":"tag41","target":"key0","value":1},{"source":"tag41","target":"key4","value":1},{"source":"tag41","target":"key5","value":1},{"source":"tag41","target":"key7","value":1},{"source":"tag41","target":"key11","value":1},{"source":"tag41","target":"key14","value":1},{"source":"tag41","target":"key26","value":1},{"source":"tag41","target":"key28","value":1},{"source":"tag41","target":"key33","value":1},{"source":"tag41","target":"key35","value":1},{"source":"tag41","target":"key36","value":1},{"source":"tag41","target":"key42","value":1},{"source":"tag43","target":"key4","value":1},{"source":"tag43","target":"key7","value":1},{"source":"tag43","target":"key26","value":1},{"source":"tag43","target":"key33","value":1},{"source":"tag44","target":"key4","value":1},{"source":"tag44","target":"key26","value":1},{"source":"tag44","target":"key33","value":1},{"source":"tag46","target":"key7","value":1},{"source":"tag46","target":"key26","value":1},{"source":"tag46","target":"key27","value":1},{"source":"tag46","target":"key45","value":1},{"source":"news","target":"tag1","value":1},{"source":"news","target":"tag2","value":1},{"source":"news","target":"tag3","value":1},{"source":"news","target":"tag6","value":1},{"source":"news","target":"tag8","value":1},{"source":"news","target":"tag10","value":1},{"source":"news","target":"tag15","value":1},{"source":"news","target":"tag17","value":1},{"source":"news","target":"tag19","value":1},{"source":"news","target":"tag20","value":1},{"source":"news","target":"tag22","value":1},{"source":"news","target":"tag24","value":1},{"source":"news","target":"tag25","value":1},{"source":"news","target":"tag29","value":1},{"source":"news","target":"tag30","value":1},{"source":"news","target":"tag31","value":1},{"source":"news","target":"tag32","value":1},{"source":"news","target":"tag34","value":1},{"source":"news","target":"tag38","value":1},{"source":"news","target":"tag41","value":1},{"source":"news","target":"tag43","value":1},{"source":"news","target":"tag44","value":1},{"source":"news","target":"tag46","value":1}]};
```

Comment:
If we do not need this, why not remove from this file? It is such a big block, and takes lots of space of this file.

Reply:
This is a sample, in case you will need it for debugging. I can delete it for you if you want ...